raiden-ts/tests/integration/integration.spec.ts

```diff
@@ -71,15 +78,55 @@ describe('integration', () => {
 
   test('account is funded', async () => {
     const balance = await raiden.getBalance(raiden.address);
-    expect(balance.eq(bigNumberify('100000000000000000000000'))).toBe(true);
-  });
-
-  test('mint tokens', async () => {
-    await expect(raiden.mint(getToken(), parseEther('1'))).resolves.toMatch('0x');
-    await expect(raiden.getTokenBalance(getToken())).resolves.toStrictEqual(parseEther('1'));
+    expect(balance.eq(bigNumberify(ethBalance))).toBe(true);
   });
 
-  test('open channel', async () => {
-    await expect(raiden.openChannel(getToken(), partner2)).resolves.toMatch('0x');
+  describe('mediated transfer', () => {
+    let routes: RaidenPaths;
+
+    test('mint TTT', async () => {
+      await expect(raiden.mint(getToken(), tttBalance)).resolves.toMatch('0x');
+      await expect(raiden.getTokenBalance(getToken())).resolves.toStrictEqual(
+        bigNumberify(tttBalance),
+      );
+    });
+
+    test('open channel with partner #1', async () => {
+      await expect(
+        raiden.openChannel(getToken(), partner1, { deposit: tttBalance }),
+      ).resolves.toMatch('0x');
+    });
+
+    test('mint SVT', async () => {
+      await expect(
+        raiden.mint(process.env.SVT_TOKEN_ADDRESS as string, svtBalance),
+      ).resolves.toMatch('0x');
+      await expect(
+        raiden.getTokenBalance(process.env.SVT_TOKEN_ADDRESS as string),
+      ).resolves.toStrictEqual(bigNumberify(svtBalance));
+    });
+
+    test('deposit to UDC', async () => {
+      await expect(raiden.depositToUDC(svtBalance)).resolves.toMatch('0x');
+      await expect(raiden.getUDCCapacity()).resolves.toStrictEqual(bigNumberify(svtBalance));
+
+      // Give PFS some time
+      await wait(3000);
+    });
+
+    test('find routes to partner #2', async () => {
+      await expect(raiden.getAvailability(partner2)).resolves.toMatchObject({ available: true });
+      routes = await raiden.findRoutes(getToken(), partner2, 50);
+      expect(routes).toMatchObject([{ fee: bigNumberify(0), path: [partner1, partner2] }]);
+    });
+
+    test('10 consecutive transfers to partner #2', async () => {
+      for (let i = 0; i < 10; i++) {
+        await expect(raiden.getAvailability(partner2)).resolves.toMatchObject({ available: true });
```

Comment:
We don't need to check for availability in the loop though right? we only need to do it the first time

Reply:
Oh yeah that was a left over from the waitTransfer() thing, will fix